prow/plugins.yaml

```diff
@@ -31,3 +31,6 @@ kubernetes:
 
 kubernetes-incubator:
 - cla
+
+kubernetes-security:
```

Comment:
Done. It doesn't make much of a difference now, but it might be better in the future.